docker/run_update_data.sh

```diff
@@ -35,3 +35,6 @@ docker-compose run processor python docker/fetch_normalization_data.py --product
 
 # Create weekly reports
 docker-compose run processor python docker/create_weekly_tables.py
+
+# Create ES indexes for the next 2 weeks
+docker-compose run processor scripts/socorro create_recent_es_indices
```

Comment:
I ran this today (Friday, January 12th, week 2), and it created the following indexes: socorro201801, socorro201802, and socorro201803. That seems right, but I think it doesn't jive with the comment here.

I think we could go vague (it's explained elsewhere) and say "Create ES indexes" or go specific and say something like "Create last week's index, this week's index, and next week's index". I think I'd go vague because then we have fewer places to update if we ever change the amount.